### ISSUE TITLE
\usetikzlibrary{feynman}

### ISSUE THREAD

**Issue by hmenke** [JP-Ellis/tikz-feynman#65]
Are you interested in making `tikz-feynman` part of the official PGF/TikZ distribution?

**Comment by JP-Ellis** [comment]
Oh, sorry I must've missed the original issue being raised.  That would be amazing to see `tikz-feynman` being incorporated into PGF/TIkZ.  The only question I have is how would development continue/resume?

I haven't had much time to work on this lately, and although I think it is pretty complete, there may be a few things I might add in the future.

**Comment by hmenke** [comment]
This issue grew out of the fact that I spotted the `patch.lua` files in your repo, so it seemed as if `tikz-feynman` needed special cooperation from PGF to work correctly, but I realized since that these are actually bugfixes.  Therefore I think it is probably wise to leave `tikz-feynman` outside the mainline repo, especially since it comes with its own publication.

**Comment by JP-Ellis** [comment]
The patched files for PKG/TikZ.  I indeed remember finding a bug a *long* time ago.  If I remember correctly, it wasn't a very extensive patch and didn't break TikZ.  I believe I tried to report it upstream, but I can't remember what came of it (as it wasn't clear at the time where TikZ is being developed).

EDIT: Glad to see TikZ development has moved to Github.  It'll be much easier for people to submit pull requests and raise issues 👍
